src/pages/onboarding/role-selection.tsx

```diff
@@ -14,8 +14,8 @@ const RoleSelectionComponent: NextPage = () => {
 };
 
 const RoleSelectionPage = withFeatureFlag(
-  RoleSelectionComponent,
+  withPageAuthRequired(RoleSelectionComponent),
```

Comment:
I realised that these checks were the wrong way round, it should check for the feature flag before redirecting the user to sign-in